File: frontend/db_connect.php
<?php
$host = "localhost";
$user = "root";  // your MySQL username
$pass = "root123";  // your MySQL password
$db = "certificate_verification";

$conn = new mysqli($host, $user, $pass, $db);

if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}
?>

File: frontend_tkinter/db_connect.py
# db_connect.py
import mysql.connector

def connect_db():
    try:
        connection = mysql.connector.connect(
            host="localhost",        # Don’t change this unless MySQL is remote
            user="root",             # Your MySQL username
            password="",# Replace with your actual MySQL password
            database="certificate_db"
        )
        return connection
    except mysql.connector.Error as err:
        print(f"❌ Database Connection Failed: {err}")
        return None
